packages/node/src/sdk.ts

```diff
@@ -152,3 +144,34 @@ export async function close(timeout?: number): Promise<boolean> {
   }
   return Promise.reject(false);
 }
+
+/**
+ * A function that returns a Sentry release string dynamically from env variables
+ */
+export function getSentryRelease(): string {
+  // Fetches the variable that sentry-webpack-plugin injects
+  const global = getGlobalObject<Window>();
+  let globalSentryRelease = undefined;
+  if (global.SENTRY_RELEASE && global.SENTRY_RELEASE.id) {
+    globalSentryRelease = global.SENTRY_RELEASE.id;
+  }
+
+  const sentryRelease = JSON.stringify(
```

Comment:
`JSON.stringify` is redundant here, as it the value is used directly and not used in webpacks env. replacements like it is in `gatsby`.

Reply:
But is not it necessary to fix this ? -> https://sentry.my.sentry.io/organizations/sentry/issues/286540/events/769bff308563499d8f807b67e787faf2/?project=4

So that if an object is entered for release, it is stringified?

Reply:
I could also possibly only stringify if it an object, I am not entirely sure that would fix the issue when Sessions are sent with release that is an object

Reply:
It's not, but you cannot pass objects as env variables, it's always a string.